### DISCUSSION TITLE
Default Admin IP 10.3.141.1  Redirecting to Octoprint Admin Screen instead of RaspAP admin Page

### DISCUSSION THREAD

**Discussion by sal562** [RaspAP/raspap-webgui#964] [Q&A]
Hi all.. just installed RaspAP on top of an existing Octopi/Klipper setup

Using raspi2 w/ AP capable WIFI Dongle.

From my understanding after install i
1) connect to the AP SSID "raspi-webgui"  Done
2) When i go to the IP http://10.3.141.1/  it routes me to the Octoprint login page and not to the RaspAP settings area

Is there another URL / IP / port number that i can use to go directly to RaspAP Admin/Settings area instead of Octopi Admin Page?

Thank you!

**Comment by billz** [comment]
Solution is to run RaspAP on an alternate web port
https://docs.raspap.com/faq.html#can-i-configure-an-alternate-port-for-raspaps-web-service

**Reply by sal562** [reply]
@billz  problem solved. 

Thank you!
